AUTHORS

```diff
@@ -405,6 +405,7 @@ answer newbie questions, and generally made Django that much better:
     Idan Melamed
     Ifedapo Olarewaju <ifedapoolarewaju@gmail.com>
     Igor Kolar <ike@email.si>
+    Ikko Ashimine <eltociear@gmail.com>
```

Comment:
Fixing a typo is not enough to add yourself to `AUTHORS`, see [docs](https://docs.djangoproject.com/en/stable/internals/contributing/writing-code/coding-style/#miscellaneous).

Reply:
@felixxm Thank you for contacting me, I deleted it.